csharp/ql/lib/semmle/code/csharp/security/dataflow/XSSQuery.qll

```diff
@@ -69,13 +67,14 @@ module PathGraph {
    * `ret -> out` is summarized as the edge `arg -> out`.
    */
   query predicate subpaths(XssNode arg, XssNode par, XssNode ret, XssNode out) {
-    DataFlow2::PathGraph::subpaths(arg.asDataFlowNode(), par.asDataFlowNode(), ret.asDataFlowNode(),
-      out.asDataFlowNode())
+    XssTracking::PathGraph::subpaths(arg.asXssDataFlowNode(), par.asXssDataFlowNode(),
+      ret.asXssDataFlowNode(), out.asXssDataFlowNode())
   }
 }
 
 private newtype TXssNode =
-  TXssDataFlowNode(DataFlow2::PathNode node) or
+  TXssDataFlowNode(DataFlow2::PathNode node) or // Deprecated
```

Comment:
I don't think we want this, as it means we basically have to compute the flow graph twice. Instead, I think for this case we should simply break backwards compatibility.

Reply:
Ok - that is fair enough.
I will do that instead and add a change node (I suppose that is breaking change which requires bumping the major version number?)